src/Storages/System/StorageSystemQueryCache.cpp

```diff
@@ -16,6 +16,7 @@ NamesAndTypesList StorageSystemQueryCache::getNamesAndTypes()
         {"key_hash", std::make_shared<DataTypeUInt64>()},
         {"expires_at", std::make_shared<DataTypeDateTime>()},
         {"stale", std::make_shared<DataTypeUInt8>()},
+        {"compressed", std::make_shared<DataTypeUInt8>()},
```

Comment:
just curious, why we don't add this column to the end ? readability is affected after result size ?

Reply:
Didn't think too much about the order :) But you are right, perhaps the important stuff (query, result_size, stale), should come first. I'd adjust that in a separate PR.